pkg/storage/s3/s3.go

```diff
@@ -76,6 +76,7 @@ func (d *driver) ConfigEnv() (envs []corev1.EnvVar, err error) {
 		corev1.EnvVar{Name: "REGISTRY_STORAGE_S3_REGION", Value: d.Config.Region},
 		corev1.EnvVar{Name: "REGISTRY_STORAGE_S3_REGIONENDPOINT", Value: d.Config.RegionEndpoint},
 		corev1.EnvVar{Name: "REGISTRY_STORAGE_S3_ENCRYPT", Value: fmt.Sprintf("%v", d.Config.Encrypt)},
+		corev1.EnvVar{Name: "REGISTRY_STORAGE_S3_KEYID", Value: fmt.Sprintf("%v", d.Config.KeyID)},
```

Comment:
why not %s ?

Reply:
also i'd skip appending this if d.Config.KeyID is empty

Reply:
in fact you probably don't need the fmt.Sprintf at all since it is a string?

Reply:
Updated in next push